### PR TITLE
Initial commit of 18F's Python Ecosystem Guide

### PR DESCRIPTION
**Pull request by ccostino**
This changeset adds the initial content of our Python Ecosystem Guide originally written in Google Docs.

### CODE REVIEW

**Comment by ccostino** [comment]
cc @jeremiak and @mbland

**Comment by jackiekazil** [comment]
@ccostino I was going to review this... but I am wondering what the "python-guide-enhancement" branch is and how it relates to "python-guide".... it looks like maybe we can combo into one PR? Yes? No?

**Comment by brittag** [comment]
@jackiekazil Oh, that's just a set of suggestions I made for this document: https://github.com/18F/dev-environment-standardization/pull/7

**Comment by jackiekazil** [comment]
@brittag  ty for the context. 
@ccostino can you merge that into this branch? (Assuming you agree.) :-)

File: pages/languages/python.md [hunk 1]
```diff
@@ -3,3 +3,264 @@ permalink: /languages/python/
...
+First, if you are using a version of Python prior to 3.4 (or older than 2.7.9
+if you are using the 2.x series), you will need to
+[install `pip` manually](http://pip.readthedocs.org/en/stable/installing/).
+In all versions of Python prior to version 3.4 you will also need to
+[install `virtualenv`](https://virtualenv.readthedocs.org/en/latest/installation.html)
+globally, as well as any `virtualenv` support tools you plan on using (e.g.,
+`virtualenvwrapper` and `autoenv`).
+
+IPython (now a sub-component of the Jupyter project) is another useful
+exception as it is a much richer REPL (Run-Eval-Print Loop) to use when
```
Comment (jmcarp) [inline pages/languages/python.md]: REPL=read eval print loop

Comment (ccostino) [inline pages/languages/python.md]: D'oh!  So it is, thanks!

**Comment by ccostino** [comment]
@jackiekazil and @brittag:  branch merged!

@jmcarp:  REPL clarification fixed (also fixed in PR for the Ruby Guide)!

**Pull request merged**